src/components/member-role-update/index.js

```diff
@@ -26,6 +26,16 @@ const MemberRoleUpdate = () => {
       setUpdateStatus('Some error occured, please contact admin');
     }
   };
+  const archiveMember = async (user) => {
+    setIsUpdating(true);
+    const { status } = await archiveMember(user);
+    setIsUpdating(false);
```

Comment:
Should we imperatively set the spinner logic, or have API state derive this data?

Reply:
Yes we can do it API state derived, and that makes more sense.

Reply:
https://github.com/Real-Dev-Squad/website-members/issues/272